cprint/cprint.py

```diff
@@ -75,15 +75,15 @@ def err(cls, arg, interrupt=False, *args, **kwargs):
             exit(1)
 
     @classmethod
-    def fatal(cls, arg, interrupt=False, *args, **kwargs):
+    def fatal(cls, arg, interrupt=False, fatal_message="Fatal error: Program stopped." *args, **kwargs):
```

Comment:
Hello, thanks for the PR!

There's a missing comma tho, so [the tests are failing](https://travis-ci.org/github/EVasseure/cprint).
[![Build Status](https://travis-ci.org/EVasseure/cprint.svg?branch=master)](https://travis-ci.org/EVasseure/cprint)

```suggestion
    def fatal(cls, arg, interrupt=False, fatal_message="Fatal error: Program stopped.", *args, **kwargs):
```